guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
 
-Let us look at the main transaction options senders need to configure when moving bitcoin:
+## Payment entry points
 
-- **Amount** — How much to send
-- **Recipient address** — Where to send the bitcoin
-- **Coin selection** — Which coins/inputs to use (optional)
-- **Fee settings** — Prioritize fast confirmation or low cost (optional)
+The need to send bitcoin can be triggered by many different use cases, and initiated by both sender and receiver. Common entry points are:
 
-You do not need to follow the order below. Feel free to tailor the configuration's order for the payment to what best serves your users. For example, you may make users set the amount before they enter the address.
+- Manual entry of address and amount
+- Sending to a previously saved contact
+- Copy & pasting payment information
+- Clicking a payment link
+- Scanning a QR code
+- Holding a device close to another device (using [NFC](https://en.wikipedia.org/wiki/Near-field_communication))
 
-## Get the recipient address
+<div class="image-slide-gallery">
 
-<div class="center" markdown="1">
-{% include image.html
-   image = "/assets/images/guide/payments/send/get-recipient-address.png"
-   retina = "/assets/images/guide/payments/send/get-recipient-address@2x.png"
-   alt-text = "Graphic showing chat messages between the sender and receiver. The sender sends a bitcoin address and the receiver agrees to pay."
-   width = 400
-   height = 400
-   layout = "float-right-desktop"
-%}
+{% for item in page.imagesEntryPoints %}
 
-To send a payment on the Bitcoin blockchain, the sender needs to obtain an address from the recipient. Since Bitcoin [addresses]({{ '/guide/glossary/address' | relative_url }}) are long and seemingly random, they are best shared by copying and pasting in plain text, as a [payment link]({{ '/guide/designing-products/wallet-interoperability/#payments' | relative_url }}), or as a scannable [QR Code]({{ '/guide/designing-products/wallet-interoperability/#qr-codes' | relative_url }}).
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
+%}
+{% endfor %}
 
-The receiver does this by generating a new address in their wallet application, then sharing it with the sender. If the sender and receiver are physically close to each other, scanning the receiver's address as a QR Code will be easy. Still, if they are not, they can send the address as text in any regular communication tool like email, SMS, etc.
 </div>
 
-## Inputting an address
+Since users cannot control how an address or invoice is presented to them, wallets should be highly flexible in terms of input options and interoperability.
```

Comment:
```suggestion
Since users cannot control how a payment request is presented to them, wallets should be highly flexible in terms of input options and [interoperability]({{ '/guide/designing-products/wallet-interoperability/' | relative_url }}). If your wallet does not support a particular payment request, users should be presented with an human readable [error]({{ '/guide/payments/send/#errors' | relative_url }}).
```

Reply:
Small typo in the suggestion, should be `a human readable` (not `an`). Would you mind updating, then I'll commit?

Reply:
Updated :)